services/webdav/pkg/service/v0/service.go

```diff
@@ -393,36 +407,37 @@ func (g Webdav) PublicThumbnailHead(w http.ResponseWriter, r *http.Request) {
 		default:
 			renderError(w, r, errInternalError(err.Error()))
 		}
-		g.log.Error().Err(err).Msg("could not get thumbnail")
+		logger.Error().Err(err).Msg("could not get thumbnail")
 		return
 	}
 
 	w.WriteHeader(http.StatusOK)
 }
 
 func (g Webdav) sendThumbnailResponse(rsp *thumbnailssvc.GetThumbnailResponse, w http.ResponseWriter, r *http.Request) {
+	logger := g.log.SubloggerWithRequestID(r.Context())
 	client := &http.Client{
 		// Timeout: time.Second * 5,
 	}
 
 	dlReq, err := http.NewRequest(http.MethodGet, rsp.DataEndpoint, http.NoBody)
 	if err != nil {
 		renderError(w, r, errInternalError(err.Error()))
-		g.log.Error().Err(err).Msg("could not download thumbnail")
+		logger.Error().Err(err).Msg("could not download thumbnail")
 		return
 	}
 	dlReq.Header.Set("Transfer-Token", rsp.TransferToken)
 
 	dlRsp, err := client.Do(dlReq)
 	if err != nil {
 		renderError(w, r, errInternalError(err.Error()))
-		g.log.Error().Err(err).Msg("could not download thumbnail")
+		logger.Error().Err(err).Msg("could not download thumbnail")
```

Comment:
@C0rby @butonic I think this could be level warn. Could the admin do anything about it when we land in this codepath?